### PR TITLE
maintenance/repositories: add signing repositories

### PR DESCRIPTION
**Pull request by Duncaen**

### CODE REVIEW

File: src/maintenance/repositories/signing.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Signing repositories
+
+Remote repositories are **must** be signed, for local repositories this is not
```
Comment (CameronNemo) [inline src/maintenance/repositories/signing.md]: typo "[repos] are must be"

Comment (Vaelatern) [reply src/maintenance/repositories/signing.md]: If we are here making changes, `local repositories do not need to be signed` is mildly more idiomatic

File: src/maintenance/repositories/signing.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The [xbps-rindex(1)](https://man.voidlinux.org/xbps-rindex.1) tool is used to
+sign repositories. First initialize the repository metadata with signing
+properties (this is only required once).
+
+```
+$ xbps-rindex --sign --signedby "I'm Groot" /path/to/repository/dir
+```
+
+Afterwards sign one or more packages with the following command, newly added
```
Comment (bobertlo) [inline src/maintenance/repositories/signing.md]: Afterwards, ...

And, if I am understanding this correctly, it maybe more clear to say:

... with the following command:
```
...
```

Any packages added in the future will also need to be signed.

Comment (Vaelatern) [reply src/maintenance/repositories/signing.md]: Don't quite agree...

Comment (Vaelatern) [reply src/maintenance/repositories/signing.md]: Actually, this makes sense, saying "Remember that this only signs the packages you already have. Newly made packages, or changed packages, will not be automatically signed."

**Comment by bobertlo** [comment]
Minus the comma, with or without the change of wording, this looks good to me.

**Pull request merged**